### PR TITLE
[Feature] Added possibility to add a explanation to a field

### PR DESCRIPTION
**Pull request by MaksymShokin**
@sofiiakvasnevska

## Issue
https://github.com/Fliplet/fliplet-studio/issues/6491

## Description
I added explanation field to all inputs except buttons, paragraph, title, and horizontal line.

## Screencast
form-exp

## Backward compatibility
This change is fully backward compatible.

## Reviewers
@upplabs-alex-levchenko

### CODE REVIEW

**Comment by MaksymShokin** [comment]
@tonytlwu @squallstar 
Fixed
https://streamable.com/yy1tp4

File: templates/components/radio.build.hbs [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <template>
+   <small v-if="description" class="help-block description">\{{ description }}</small>
```
Comment (tonytlwu) [inline templates/components/radio.build.hbs:2]:
```suggestion
  <small v-if="description" class="help-block description">\{{ description }}</small>
```

Comment (MaksymShokin) [reply templates/components/radio.build.hbs]: Fixed

File: templates/components/date.build.hbs [hunk 2]
```diff
@@ -1,3 +1,4 @@
+<small v-if="description" class="help-block description">\{{ description }}</small>
```
Comment (tonytlwu) [inline templates/components/date.build.hbs:1]: I prefer to put this inside a block element, e.g. `div`. If you want to use `small`, put the `small` tag inside a block element.

This way, the text by default will form a block on top of the input fields and makes it easier to style and adjust layouts for if necessary. In contrary, adding a block container later in the future may cause more problem.

Comment (MaksymShokin) [reply templates/components/date.build.hbs]: @tonytlwu 
In that case we do not even need small tag.

**Comment by tonytlwu** [comment]
@MaksymShokin Can you share the latest screenshot for how this looks? The original screenshot shows the explanation text under the fields instead of above them.

**Comment by MaksymShokin** [comment]
@tonytlwu 
https://streamable.com/yy1tp4

**Pull request merged**